### ISSUE TITLE
Install validation terminates with error

### ISSUE THREAD

**Issue by egulacti** [apprenda/kismatic#105]
I am using Ubuntu Xenial and Kismatic repo is added to the machine. I have package installation  enabled in the plan file. When I execute **kismatic install validate** I receive the following result. 

```
Reading installation plan file "kismatic-cluster.yaml"                          [OK]
Validating installation plan file                                               [OK]
1/2....  Configure Cluster Prerequisites                                        [OK]
2/2....  Run Cluster Pre-Flight Checks
=> The following checks failed on "192.168.20.222":
    Package Available: kismatic-etcd 1.4.5-1: kismatic-etcd 1.4.5-1 is not installed but is available in a package repository

=> The following checks failed on "192.168.20.223":
    Package Available: kismatic-kubernetes-node 1.4.5-1: kismatic-kubernetes-node 1.4.5-1 is not installed but is available in a package repository

=> The following checks failed on "192.168.20.221":
    Package Available: kismatic-kubernetes-master 1.4.5-1: kismatic-kubernetes-master 1.4.5-1 is not installed but is available in a package repository

error running preflight: error running playbook: error running ansible: exit status 2
```

**Comment by dkoshkin** [comment]
Do you ming sharing your plan file (removed password and any other sensitive info) thanks.

**Comment by egulacti** [comment]
My plan file is attached.
[kismatic-cluster-yaml.txt](https://github.com/apprenda/kismatic/files/605632/kismatic-cluster-yaml.txt)

**Comment by alexbrand** [comment]
I suspect that what you ran into is that you enabled package installation after an initial execution of `kismatic install validate`. We have a bug where we are not shutting down the inspector after it runs, which would have resulted in the inspector thinking package installation was disabled.

The `always` block in the inspector play is not being executed due to the `any_errors_fatal: true` directive in `_preflight.yaml`.

@dkoshkin Do you recall why this was added?

**Comment by alexbrand** [comment]
It's unclear if this is a bug in ansible. Opened https://github.com/ansible/ansible/issues/18602 to get some feedback from ansible team.

**Comment by alexbrand** [comment]
Ansible team is considering this a bug.

**Comment by dkoshkin** [comment]
Closed with #149
